packages/website/src/features/Packages/FunctionInput.tsx

```diff
@@ -31,10 +31,6 @@ export const FunctionInput: FC<{
     setDataArray(dataArray.splice(index, 1));
   };
 
-  useEffect(() => {
-    valueUpdated(dataArray.map((item) => item.val));
-  }, [dataArray]);
-
```

Comment:
This one was causing that default values were not parsed as expected on first render of the function input, getting wrong params values (into an array), this before the fix:

<img width="420" alt="Screenshot 2024-04-06 at 11 07 27 AM" src="https://github.com/usecannon/cannon/assets/3952453/b9bfc997-b4fe-4d65-9e20-a1259d50f8f3">

With this removal, here the right params values

<img width="639" alt="Screenshot 2024-04-06 at 11 08 07 AM" src="https://github.com/usecannon/cannon/assets/3952453/146067e2-810b-4af8-835e-e4ea08014452">